node/engine/engine.go

```diff
@@ -863,11 +864,6 @@ func (e *Engine) checkError(err error) {
 			}
 		}
 
-		<-time.After(1000 * time.Millisecond) // We wait for a bit so the previous log line has time to complete
```

Comment:
Does this line still have a purpose? Does this help us get more debugging information in logs? Maybe @lalexgap or @geoknee remeber?

Reply:
It looks like a `Panic` level log by zerolog will ensure that the log gets written, then will call `panic()`:
```
		e.logger.Panic().Msg(err.Error())
```

Reply:
Let's use that, then :-)